setup.py

```diff
@@ -42,6 +42,12 @@
         'testing': [
             'psycopg2>=2.6',
         ],
+        'pontoon': [
+            'polib==1.1.0',
+            'pygit2==0.28.2',
+            'gitpython==3.0.2',
+            'toml==0.10.0',
```

Comment:
Pinned requirements are going to cause issues for apps using this.  Could you use ranges instead?  (like `polib == 1.1.*` or `pygit2 ~= 0.28.2` — both forms allow an app to pin a newer micro version (1.1.1, 0.28.3), the second form specifies a minimum required micro)

Reply:
Will change this. Thanks for pointing it out!

Reply:
Fixed (squashed into original commit)